src/DefaultPluginStarter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace StellarWP\Telemetry;
+
+class DefaultPluginStarter extends PluginStarter {
+
+	public function __construct( array $args = [] ) {
+		$args = wp_parse_args( $args, [
+			'activation_hook'      => new DefaultActivationHook(),
+			'optin_template'       => new DefaultOptinTemplate(),
+			'plugin_slug'          => 'stellarwp-telemetry-starter',
+			'plugin_version'       => '1.0.0',
+			'option_to_inherit'    => 'freemius_inherit',
```

Comment:
Are we using freemius as the default to inherit or should there be no default? @ChrisMKindred